src/cogent3/app/align.py

```diff
@@ -650,6 +679,20 @@ def __init__(self, equifreq_mprobs=True):
         ----------
         equifreq_mprobs : bool
             If true, specifies equally frequent motif probabilities.
+
+        Examples
+        --------
+
+        Create a sample alignment and compute the Information Content alignment
+        quality score. By default, equally frequent motif probabilities are
```

Comment:
Change "By default, equally..." to "The default is..." and remove the "specified with..." remainder of the setnence.